src/v2/view-builder/views/ov/SelectEnrollmentChannelOktaVerifyView.js

```diff
@@ -13,8 +13,14 @@ const Body = BaseForm.extend({
     const schemas = BaseForm.prototype.getUISchema.apply(this, arguments);
     // filter selected channel
     const channelField = _.find(schemas, (schema) => schema.name === 'authenticator.channel');
-    channelField.options = _.filter(channelField?.options, (option) =>
-      option.value !== this.options.appState.get('currentAuthenticator')?.contextualData?.selectedChannel);
+    if (BrowserFeatures.isAndroid() || BrowserFeatures.isIOS()) {
```

Comment:
implementing similar approach as [this](https://github.com/okta/okta-signin-widget/blob/e703b3a1d25d72175d9cc65acb37f6e39c7a5b71/src/v2/models/AppState.ts#L280)

Reply:
As per [this](https://github.com/okta/okta-signin-widget/blob/6262eecec7273684223dad565fb25b49a91d401f/src/util/BrowserFeatures.ts#L77) , windows phone UA contains android in them. So just using the existing method.

Reply:
Perfect! I remember this now 😅 Ty for checking!